.github/workflows/go.yml

```diff
@@ -39,3 +39,11 @@ jobs:
       - name: Check go mod is tidy
         run: make check-tidy
 
+      - name: Install Sysl and Arr.ai
+        run: |
+          go get -u -v github.com/anz-bank/sysl/cmd/sysl
+          go get -u -v github.com/arr-ai/arrai/cmd/arrai
```

Comment:
We need a good way of tracking what versions of `sysl` and `arrai` are required for these transforms.